acos_client/v30/file/ssl_cert.py

```diff
@@ -53,16 +53,16 @@ def _set(self, file=None, cert=None, size=None, certificate_type=None,
         return self._post(self.url_prefix + file, file_name=obj_params["file"],
                           file_content=cert, **kwargs)
 
-    def create(self, file=None, cert=None, size=None, certificate_type=None,
-               action=None, update=False, **kwargs):
+    def create(self, file="", cert="", size="", certificate_type="", action="",
+               update=False, **kwargs):
         if self.exists(file):
             raise acos_errors.Exists
 
         self._set(file, cert, size, certificate_type, action, update=False,
                   **kwargs)
 
-    def update(self, file=None, cert=None, size=None, certificate_type=None,
-               action=None, update=False, **kwargs):
+    def update(self, file="", cert="", size="", certificate_type="", action="",
+               update=False, **kwargs):
```

Comment:
I know it's always been this way, but why have False as a default parameter value when it gets overridden in the call?